### ISSUE TITLE
update .htaccess files to be compatible with Apache 2.4

### ISSUE THREAD

**Issue by nwong212** [collectiveaccess/providence#1361]
[System requirements](https://manual.collectiveaccess.org/providence/user/setup/systemReq.html) say Apache 2.4 is a requirement, but the htaccess files are not compatible.

per https://httpd.apache.org/docs/2.4/upgrading.html

> Directives that control how authorization modules respond when they don't match the authenticated user have been removed: This includes AuthzLDAPAuthoritative, AuthzDBDAuthoritative, AuthzDBMAuthoritative, AuthzGroupFileAuthoritative, AuthzUserAuthoritative, and AuthzOwnerAuthoritative. These directives have been replaced by the more expressive [RequireAny](https://httpd.apache.org/docs/2.4/mod/mod_authz_core.html#requireany), [RequireNone](https://httpd.apache.org/docs/2.4/mod/mod_authz_core.html#requirenone), and [RequireAll](https://httpd.apache.org/docs/2.4/mod/mod_authz_core.html#requireall).

All .htaccess files in 1.7.16 use Order/Allow/Deny instead of Require.

For some server configurations, this can cause a 500 internal server error.

**Comment by collectiveaccess** [comment]
They're compatible on every 2.4 system I've seen. What error(s) exactly are yours throwing?

**Comment by nwong212** [comment]
i get a 500 error with server error_log message:
>Invalid command 'Order', perhaps misspelled or defined by a module not included in the server configuration

updating the htaccess fixes the 500 error, and allows me to do the database install step, but now theme files (all images, css, scripts) do not show up and appear to be called using the server file paths (/home/public/file).

**Comment by nwong212** [comment]
the theme files error turns out to be because i'm using a shared server. I had to set `__CA_URL_ROOT__` to `""`. So far updating all .htaccess files to `Require all granted` or `Require all denied` is solving the main error 500 issue for my server set up.

**Comment by goetzk** [comment]
I saw this report while looking at another problem 

I noticed mod_access_compat (https://httpd.apache.org/docs/2.4/mod/mod_access_compat.html) was created with 2.4 to handle the previous syntax. Its enabled by default on Ubuntu and RHEL8 but could conceivably not be in all cases - eg a shared hosting environment.

I suspect unrelated to this issue, but a possible red herring for it, is ensuring AllowOverride (https://httpd.apache.org/docs/2.4/mod/core.html#allowoverride) is enabled in your websites vhost so the htaccess files can actually do their job.

Hopefully these clues might be useful to the next person who stumbles over this issue.
